app/assets/javascripts/components/CreateCase.jsx

```diff
@@ -0,0 +1,213 @@
+class CreateCase extends React.Component {
+    constructor(props) {
+        super();
+        this._handleChange = this._handleChange.bind(this);
+        this._handleSubmit = this._handleSubmit.bind(this);
+        this.state = {
+            showForm: true,
+            message: '',
+            error: ''
+        };
+    }
+
+    _handleChange = (e) => {
+        this.setState({ [$(e.target).attr("name")] : $(e.target).val() });
+    }
+
+    _handleSubmit = (e) => {
+        e.preventDefault();
+        const payload = {
```

Comment:
the standard way of enforcing parameters when sending this payload here is 

```javascript
const payload = {
  case: {
    // All of your stuff in here
  }
}
```

And then see my comment down below about strong parameters in your rails controller for cases